parsers/pcap.go

```diff
@@ -315,7 +316,7 @@ func conjoinRequestAndResponse(reqCh chan *http.Request, resCh chan *http.Respon
 				}
 
 				// discard HTTP body
-				n, err := io.CopyBuffer(io.Discard, res.Body, copyBuf[:])
+				n, err := io.CopyBuffer(ioutil.Discard, res.Body, copyBuf[:])
```

Comment:
@tkuchiki Is it needed testing on go 1.14 and go 1.15?

My understanding is that most users will be using pre-built executables, and there will be few situations where it is necessary to build with the old Go.

However, it would be preferable to make this change together with CI settings and other functions such as ReadAll at once, so we have aligned them to use ioutil nowadays.

Reply:
@karupanerura No, it isn't.
Let's drop the older versions!

Reply:
OK! fixed at 263a4ec

Reply:
Awesome, thanks!